addons/website/static/src/components/dialog/page_properties.js

```diff
@@ -114,6 +114,10 @@ export class DuplicatePageDialog extends Component {
     }
 
     async duplicate() {
+        // TODO In master support multiple pages.
```

Comment:
What do you mean support multi page? It's already working no? I can select multiple pages and clone them and they will correctly be duplicated with a unique URL and unique name (name could end up with same name as another url already existing but it's not a problem to have same name)

Or do you mean, having `duplicate` work out of the box here if `pageIds` is provided?

Couldn't we already do it in stable?

Eg: inside the `if (this.state.name)`: loop through the provided ids (maybe in stable concat pageId and pageIds, but in master keep only pageIds) and clone, then move `this.website.goToWebsite({path: res, edition: true});` inside a condition like "if (!pageIds)" to keep the same behavior when not calling it with the new param.
And in master, move that in a default behavior of `onDuplicate` maybe? So you would still have to override it here to reload the list view and not going to the website.

That would avoid having to copy the orm call to clone_page, but it's maybe not that bad, ignore this if you want, I'll let you judge if it's worth it / better. Ignoring this comment is very fine.

Reply:
Yes, I meant "work out of the box here if pageIds is provided". I'll do what you suggest, and get rid of the override property in stable.

Reply:
FYI - I did instead mention that `pageIds` overrides `pageId`... it is a bit weird for the caller to have to extract the first id from its list so that it gets concatenated. But since `pageId` is mandatory, a value has to be provided anyway.